packages/components/src/components/hds/popover-primitive/index.ts

```diff
@@ -32,16 +32,64 @@ export {
   DEFAULT_PLACEMENT,
 } from '../../../modifiers/hds-anchored-position.ts';
 
-export default class HdsPopoverPrimitiveComponent extends Component {
+import type { FloatingUIOptions } from '../../../modifiers/hds-anchored-position.ts';
+import type { ModifierLike } from '@glint/template';
+
+export interface HdsPopoverPrimitiveSignature {
+  Args: {
+    isOpen: boolean;
```

Comment:
we probably want to mark some of this as optional

Reply:
you're right, fixed in https://github.com/hashicorp/design-system/pull/2105/commits/931642fc2ee9ec2ea17fd1087dd0541d90ef3472